corpus.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var request = require('request');
+var lunr = require('lunr');
+
+function Corpora(name, baseurl, indexPath, remote) {
+	
+	this.name = name;
+	this.baseurl = baseurl;
+	this.indexPath = indexPath;
+	this.remote = remote;
+	this.key = name.toLowerCase().split(' ').join('-');
```

Comment:
You already have a `key` property specified in `corpora-config.json` so no need to create it here. Or alternatively create it here but don't specify it in `corpora-config.json`.